app/bundles/CoreBundle/EventListener/DashboardSubscriber.php

```diff
@@ -97,7 +97,7 @@ public function onWidgetDetailGenerate(WidgetDetailEvent $event)
                     if (!empty($log['bundle']) && !empty($log['object']) && !empty($log['objectId'])) {
                         try {
                             $model = $this->modelFactory->getModel($log['bundle'].'.'.$log['object']);
-                            $item  = $model->getEntity($log['objectId']);
+                            $item  = $model->getEntity($log['objectId']) ?: '';
                             if (method_exists($item, $model->getNameGetter())) {
```

Comment:
I'd rather suggest this:
```suggestion
                            $item  = $model->getEntity($log['objectId']);
                            if ($item && method_exists($item, $model->getNameGetter())) {
```
The first argument to [method_exists](https://www.php.net/manual/en/function.method-exists.php) should be an object or a class. An empty string might be deprecated in the future PHP versions too.